src/main/resources/application.properties

```diff
@@ -2,9 +2,11 @@ quarkus.openshift.route.expose=true
 quarkus.openshift.labels."app"=wildfly-bot
 quarkus.openshift.env.secrets=wildfly-bot
 quarkus.openshift.resources.requests.cpu=100m
-quarkus.openshift.resources.requests.memory=200Mi
+quarkus.openshift.resources.requests.memory=192Mi
```

Comment:
do we need 192Mi when we max heap on 128M? Not sure about the numbers.

Reply:
those should be 2 different numbers. Where 128Mi is just for the heap itself, which JVM operates upon. 192Mi is for pod as a whole